package.json

```diff
@@ -33,12 +33,14 @@
     "types:storefront": "yarn workspace storefront types",
     "version-packages": "changeset version",
     "publish": "yarn build && changeset publish",
-    "chromatic": "yarn workspace @designsystemet/storybook chromatic"
+    "chromatic": "yarn workspace @designsystemet/storybook chromatic",
+    "api-report:cli": "yarn workspace @digdir/designsystemet api-report"
```

Comment:
Maybe call this `report:cli-api`? 

We can then have a nice naming convention for other stuff down the line, `report:test`, `report:tokens`, `report:something-seomthing`.

Reply:
I don't have strong opinions on this, so I changed it. Easy enough to change to something else if needed :)